content/en/cloud_cost_management/saas_costs.md

```diff
@@ -256,6 +256,22 @@ Your Twilio cost data for the past 15 months can be accessed in Cloud Cost Manag
 
 [101]: https://app.datadoghq.com/integrations/twilio
 
+{{% /tab %}}
+{{% tab "GitHub" %}}
+
+1. Create a personal authorization token (classic), with the `manage_billing:enterprise` and `repo` scopes on the [Personal Access Tokens][101] page in GitHub.
```

Comment:
```suggestion
1. Create a personal authorization token (classic), with the `manage_billing:enterprise` and `repo` scopes on the [Personal Access Tokens][101] page in GitHub.
```
This link to Personal Access Tokens isn't rendering, you'll have to add a line within the Tabs section such as

[101]: Personal Access Tokens link